README.md

```diff
@@ -41,6 +44,21 @@ const translated: Transcript = await supadata.youtube.translate({
   lang: 'es',
 });
 
+// Get YouTube video details
+const video: YoutubeVideo = await supadata.youtube.video('dQw4w9WgXcQ');
+
+// Get YouTube channel details
+const channel: YoutubeChannel = await supadata.youtube.channel.get('CHANNEL_ID');
+
+// Get YouTube playlist details
+const playlist: YoutubePlaylist = await supadata.youtube.playlist.get('PLAYLIST_ID');
+
+// Get channel videos (with optional limit)
+const channelVideos: string[] = await supadata.youtube.channel.videos('CHANNEL_ID', 10);
+
+// Get playlist videos (with optional limit)
+const playlistVideos: string[] = await supadata.youtube.playlist.videos('PLAYLIST_ID', 10);
+
```

Comment:
1. The existing of the Youtube functions accept an object with parameters. I'd like to stick to this pattern instead of having multiple function parameters.

2. Also, would be great to follow the existing convention and enter specific values instead of "CHANNEL_ID" - it's more friendly for the user who wants to quickly test the library

3. Again a "getter" like `supadata.youtube.playlist.get` popped in, just like in the last PR. That's not necessary and deviates from convention

PS - the OpenAPI spec has code snippets for all these functions so I don't understand why you decided to use them